### ISSUE TITLE
About the initial learning rate of RPN in FPN

### ISSUE THREAD

**Issue by funnyzhou** [CharlesShang/FastMaskRCNN#10]
In the paper, the initial learning rate of RPN is 0.02. I've implemented the RPN following the instructions in the paper using caffe. However, when I set the lr to 0.02, the network just cannot converge. **And when I set the lr to 0.001, the learning process seems to be too slow to catch up the performance of the original RPN.** 

So, how about this version in tf? Have you ever tried a larger lr than 0.001?

**Comment by 1292765944** [comment]
Do you use the 800x1200 input? Meanwhile, what the sampling strategy of your anchor layer, do you sample anchors from those outside the image for training? How many gpus and effective batchsizes do you use in training?

**Comment by funnyzhou** [comment]
1. The shorter edge is 800. 
2. The same strategy in the FPN paper.
3. I used only one gpu and the batch size is 256.

**Comment by 1292765944** [comment]
@funnyzhou what the iter_size you use. Since you use one gpu, you can try large iter_size

**Comment by funnyzhou** [comment]
I tried iter_size = 8, 16. However, it didn't work...
